src/App.tsx

```diff
@@ -1,24 +1,144 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Remove this unnecessary ESLint disable rule comment, as it does not apply to this file.